api/src/middlewares/services/EmailService/NotifyService.ts

```diff
@@ -70,7 +70,7 @@ export class NotifyService implements EmailServiceProvider {
     };
   }
 
-  getPersonalisationForTemplate(answers: AnswersHashMap, reference: string, paid: boolean, template: Record<string, string | boolean>) {
+  getPersonalisationForTemplate(answers: AnswersHashMap, reference: string, paid: boolean, template: Record<string, string | boolean | undefined>) {
```

Comment:
should template be typeof standard? or should it just be a generic type? 

```
type PersonalisationTemplate = {
  [Property in keyof SomeOtherType]: string | boolean | undefined;
}
```
?